src/utils/defaultConfig.ts

```diff
@@ -0,0 +1,22 @@
+export default {
+  plugins: {
+    'yaml-syntax': true,
+    'open-policy-agent': true,
+    'resource-links': true,
+    'kubernetes-schema': true,
+    annotations: true
+  },
+  rules: {
+    'yaml-syntax/no-bad-alias': 'warn',
+    'yaml-syntax/no-bad-directive': false,
+    'open-policy-agent/no-last-image': 'err',
+    'open-policy-agent/cpu-limit': 'err',
+    'open-policy-agent/memory-limit': 'err',
+    'open-policy-agent/memory-request': 'err'
+  },
+  settings: {
+    "kubernetes-schema": {
+      schemaVersion: "v1.24.2",
```

Comment:
suggest to at least use 1.26.8 - https://kubernetes.io/releases/ - guessing that 1.24 has been EOL:d!?

Reply:
Good point, I guess it will be good to update framework related configs too 👍